src/init.cpp

```diff
@@ -405,7 +405,7 @@ void SetupServerArgs()
     gArgs.AddArg("-maxtimeadjustment", strprintf("Maximum allowed median peer time offset adjustment. Local perspective of time may be influenced by peers forward or backward by this amount. (default: %u seconds)", DEFAULT_MAX_TIME_ADJUSTMENT), false, OptionsCategory::CONNECTION);
     gArgs.AddArg("-maxuploadtarget=<n>", strprintf("Tries to keep outbound traffic under the given target (in MiB per 24h), 0 = no limit (default: %d)", DEFAULT_MAX_UPLOAD_TARGET), false, OptionsCategory::CONNECTION);
     gArgs.AddArg("-onion=<ip:port>", "Use separate SOCKS5 proxy to reach peers via Tor hidden services (default: -proxy)", false, OptionsCategory::CONNECTION);
-    gArgs.AddArg("-onlynet=<net>", "Only connect to nodes in network <net> (ipv4, ipv6 or onion)", false, OptionsCategory::CONNECTION);
+    gArgs.AddArg("-onlynet=<net>", "Actively connect to remote nodes only through network <net> (ipv4, ipv6, onion or tor), where onion and tor are synonymous. Passive, incoming connections are not affected by this option. This option can be specified multiple times.", false, OptionsCategory::CONNECTION);
```

Comment:
Please only mention `onion` here, `tor` is deprecated.

Reply:
OK, removed mentioning of "tor" and added a comment in the sourcecode, respectively.